clients/fides-js/src/components/ConsentBanner.tsx

```diff
@@ -9,8 +9,10 @@ interface BannerProps {
   experience: ExperienceConfig;
   onClose: () => void;
   bannerIsOpen: boolean;
-  children: ComponentChildren;
+  children?: ComponentChildren;
```

Comment:
can you add a docstring here to explain where in a consent banner `children` would render? if you do a 

```
/** children will render... */
children: ComponentChildren;
```

then the docstring should be interpreted nicely by IDEs